mandatory/4-tax.js

```diff
@@ -17,7 +20,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  return `£ ${(price + (price * 20) / 100).toFixed(2)}`;
```

Comment:
Well done, this works however I think you're overcomplicating things! You've already written the function that calculates the sales tax value so you can use that instead of rewriting it out!

Something like:

return "£" + calculateSalesTax(price);